### PR TITLE
Feature/#320 change select title for origin collection and destination collection in edit relation.component

### LINKED ISSUE #320: Change selectTitle for origin collection and destination collection in edit-relation.component
Image

As you can see, now the text inside the "select" is select origin table and select destination table, you have to change the word table to collection.

Create a const outside the component , on line 9 in the file `src/pods/edit-relation.component.tsx` with name:
```
const DROPDOWN_ORIGIN_TABLE_TITLE= "Select origin collection"
```
, other for destination table, and for table fields.

### PR DESCRIPTION
**Pull request by evadadev**
Close #320

### CODE REVIEW

File: src/common/components/forms/dropdown-formik/dropdown-formik.component.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ interface DropDownFormikProps {
   value?: DropdownOptionVm;
   name: string;
   options: DropdownOptionVm[];
+  // selectTitle?: string;
   selectTitle?: string;
```
Comment (brauliodiez) [inline src/common/components/forms/dropdown-formik/dropdown-formik.component.tsx:11]: In line ten there's a commented line of code

```ts
// selectTitle? :string
```

Should it be removed?

File: src/pods/edit-relation/edit-relation.component.tsx [hunk 2]
```diff
@@ -51,7 +53,7 @@ export const EditRelationComponent: React.FC<Props> = props => {
         name="toTableId"
         label="Destination Collection"
         options={tablesNameOptions}
-        selectTitle="Select destination table"
+        selectTitle={DROPDOWN_ORIGIN_TABLE_TITLE}
```
Comment (brauliodiez) [inline src/pods/edit-relation/edit-relation.component.tsx:56]: Should be destination?

**Comment by evadadev** [comment]
@brauliodiez,  close #355

**Comment by evadadev** [comment]
@brauliodiez close #355

**Review by deletidev — APPROVED** [review]

**Pull request merged**